projects/iucn/src/app/iucn.module.ts

```diff
@@ -43,6 +44,7 @@ export function createLoggerLoader(loggerApi: LoggerApi): ILogger {
   imports: [
     GraphQLModule,
     AppComponentModule,
+    LocaleModule,
```

Comment:
Iirc linter complained about `LocaleEnComponent` etc. being declared in multiple modules (I didn't want to unnecessarily import AppComponentModule to the bird atlas project) so that's why I added the `LocaleModule` as an intermediary. `kerttu-global` seems to work just fine without it even though I somehow can't find at all where the `LocaleEnComponent` declaration is coming to that project...

Reply:
Little strange but I guess it's fine if it works